scripts/site/_site/doc/app/components-overview/components-overview.component.html

```diff
@@ -0,0 +1,68 @@
+<h1>{{ language === 'en' ? 'Components Overview' : '组件总览' }}</h1>
+
+<section class="markdown">
+  <p *ngIf="language === 'en'">
+    <span nz-typography><code>ng-zorro-antd</code></span>
```

Comment:
```suggestion
    <span><code>ng-zorro-antd</code></span>
```
这里不需要 nz-typography 组件